test/CMakeLists.txt

```diff
@@ -900,6 +900,9 @@ set(DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS
     ${DYNAMIC_IMPLICITGEMM_COMMON}
     MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvAsmImplicitGemmGTCDynamicWrwXdlops)
 
+set(CK_IMPLICITGEMM_COMMON
+    MIOPEN_FIND_MODE=normal)
+
```

Comment:
[Recommendation] This is not used. Not a big harm, but I would better remove this.

Reply:
Searched and indeed not used anywhere, thus removed.